starlette/datastructures.py

```diff
@@ -246,7 +270,7 @@ def __init__(
         if kwargs:
             value = (
                 ImmutableMultiDict(value).multi_items()
-                + ImmutableMultiDict(kwargs).multi_items()
+                + ImmutableMultiDict(kwargs).multi_items()  # type: ignore[operator]
```

Comment:
Why is not a valid operation?

Reply:
this could be fixed with a bit of juggling of how this is handled, but I wanted to do this with the least amount of runtime changes necessary, so adding a `type: ignore` for now and maybe doing a rework in a future PR seems like the best bet